csrankings-m.csv

```diff
@@ -57,6 +57,7 @@ Maarten Jansen,Université libre de Bruxelles,http://homepages.ulb.ac.be/~majans
 Maarten Löffler,Utrecht University,http://www.staff.science.uu.nl/~loffl001/research.html,u-pn2g0AAAAJ
 Maarten Marx,University of Amsterdam,http://www.uva.nl/profiel/m/a/m.j.marx/m.j.marx.html,H54oRlIAAAAJ
 Maarten de Rijke,University of Amsterdam,https://staff.fnwi.uva.nl/m.derijke,AVDkgFIAAAAJ
+Maarten Sap,Carnegie Mellon University,https://maartensap.com/,gFN4QUYAAAAJ

```

Comment:
Web page indicates part-time affiliation at Allen Institute. Please confirm that CMU appointment is full-time (>= 75%).

Reply:
Hi, yes Maarten is full-time at CMU (only 20% at Allen Institute). Thanks!